### PR TITLE
Add data analysis code

### PR DESCRIPTION
**Pull request by TempixTL**

### CODE REVIEW

File: data_analysis/main.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# This file is best run as an interactive Jupyter notebook
```
Comment (timmy-ch) [inline data_analysis/main.py:1]: Perhaps a better title to the file? This is a bit vague

File: data_analysis/main.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+# %% import libraries
+import pandas as pd
+
+# %% read food data
+food = pd.read_csv('../data/fdc/food.csv').set_index('fdc_id')
```
Comment (timmy-ch) [inline data_analysis/main.py:7]: also might be worth writing the directory as a separate variable s.t. all other users know where to point the data (or at least know to change it)

File: data_analysis/main.py (hunk 1 above)
Comment (TempixTL) [reply data_analysis/main.py]: Updated in https://github.com/googleinterns/amaranth/pull/1/commits/4a1d5e31659e9491f60ea048e80320ece98369e5

File: data_analysis/main.py (hunk 2 above)
Comment (TempixTL) [reply data_analysis/main.py]: Added in https://github.com/googleinterns/amaranth/pull/1/commits/3e909e93b9fa8643c9f4a34c15fd62cf242114a5

**Pull request merged**